flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/workflow/scheduler/EmbeddedQuartzScheduler.java

```diff
@@ -271,11 +271,6 @@ public void deleteScheduleWorkflow(String workflowName, String workflowGroup)
         JobKey jobKey = JobKey.jobKey(workflowName, workflowGroup);
         lock.writeLock().lock();
         try {
-            String errorMsg =
-                    String.format(
-                            "Failed to delete a non-existent quartz schedule job: %s.", jobKey);
-            checkJobExists(jobKey, errorMsg);
-
             quartzScheduler.deleteJob(jobKey);
```

Comment:
Do you verify whether  `quartzScheduler.deleteJob` throws an exception when deleting a not exist job?

Reply:
Yes, In the testDropMaterializedTableWithDeletedRefreshWorkflowInFullMode test, even if the workflow does not exist, `quartzScheduler.deleteJob(jobKey)` will return normally without error.